assets/lang/en.json

```diff
@@ -116,6 +116,12 @@
     "pages.devices.connection.done": "Done",
     "pages.devices.connection.ok": "OK",
     "pages.devices.connection.settings": "Settings",
+    "pages.devices.connection.bluetooth_authorization.title": "Authorize app",
+    "pages.devices.connection.bluetooth_authorization.message": "This app needs authorization to pair with Bluetooth. Please authorize the app following the steps shown in the photo below.",
+    "pages.devices.connection.enable_bluetooth.title": "Enalbe bluetooth",
+    "pages.devices.connection.enable_bluetooth.message": "Bluetooth needs to be turned on. Below, you will find a screenshot with instructions on how to do that.",
+    "pages.devices.connection.disconnect_bluetooth.title": "Disconnect bluetooth",
+    "pages.devices.connection.disconnect_bluetooth.message": "Are you sure you want to disconnect the current device?",
```

Comment:
Could add the name of the device.
Instead of using the whole string just do "Are you sure you want to disconnect the ", and then when using it do (written in github so don't judge if i don't write 100% correct):
`"$locale.translate('pages.devices.connection.disconnect_bluetooth.message $device.name?"`